pkgs/development/tools/vulkan-validation-layers/default.nix

```diff
@@ -1,23 +1,27 @@
 { stdenv, fetchFromGitHub, cmake, writeText, python3
-, vulkan-headers, vulkan-loader, glslang
+, vulkan-headers, vulkan-loader, glslang, spirv-headers
 , pkgconfig, xlibsWrapper, libxcb, libXrandr, wayland }:
 
 stdenv.mkDerivation rec {
   pname = "vulkan-validation-layers";
-  version = "1.2.131.2";
+  version = "1.2.141.0";
 
   src = fetchFromGitHub {
     owner = "KhronosGroup";
     repo = "Vulkan-ValidationLayers";
     rev = "sdk-${version}";
-    sha256 = "1sz0388cr018ldx6ziplvk4v3zbg44pww77kv6kv5wxl69plwfcn";
+    sha256 = "1yfas7q122kx74nbjk3wxlyacysgncvlvq081a5dp238m88vkmbj";
   };
 
-  nativeBuildInputs = [ pkgconfig cmake python3  ];
+  nativeBuildInputs = [ pkgconfig cmake python3 spirv-headers ];
```

Comment:
The fact it's only the headers needed, doesn't justify putting it in native IMO.

Reply:
Done